website/source/docs/agent/caching/index.html.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: "docs"
+page_title: "Vault Agent Caching"
+sidebar_title: "Caching"
+sidebar_current: "docs-agent-caching"
+description: |-
+  Vault Agent Caching allows client-side caching of responses containing newly
+  created tokens and responses containing leased secrets generated off of these
+  newly created tokens.
+---
+
+# Vault Agent Caching
+
+Vault Agent Caching allows client-side caching of responses containing newly
+created tokens and responses containing leased secrets generated off of these
+newly created tokens. The renewals of the secrets that are cached are also
+managed by the agent.
+
+## Caching and Renewals
+
+Caching and renewals are managed by the agent only under these scenarios.
+
+1. Token creation requests are made through the agent and not directly to the
+   Vault server. Login endpoints from various auth methods also fall under this
+   category along with token creation endpoints of the token auth method.
+
+2. Leased secret creation requests are made through the agent using tokens that
+   are already managed by the agent.
+
+## Using Auto-Auth Token
+
+The requests that reach the agent, if they don't already bear a token as part
```

Comment:
Slightly reworded: "If the requests that reach the agent don't already bear a token as part of the request, the auto-auth token can be put to use."